packages/jest-cli/src/FailedTestsCache.js

```diff
@@ -41,8 +41,7 @@ export default class FailedTestsCache {
     if (!this._enabledTestsMap) {
       return globalConfig;
     }
-    // $FlowFixMe Object.assign
-    const newConfig: GlobalConfig = Object.assign({}, globalConfig);
+    const newConfig: GlobalConfig = {...globalConfig};
```

Comment:
do we still need to annotate it?

Reply:
Slightly worse error:
![image](https://user-images.githubusercontent.com/1404810/51135935-8715ca00-183b-11e9-8a98-c53074fa74a3.png)

vs with explicit typing:
![image](https://user-images.githubusercontent.com/1404810/51135964-9d238a80-183b-11e9-9bda-63a73b57f1aa.png)

worth it, right?

Reply:
Then I guess we can leave it :)